images/nginx/loris.Dockerfile

```diff
@@ -1,3 +0,0 @@
-FROM nginx:1.15.8-alpine
-
-COPY loris.nginx.conf /etc/nginx/nginx.conf
```

Comment:
I've moved this into the Loris repo: https://github.com/wellcomecollection/loris-infrastructure/tree/main/nginx

We're never going to use it again so it shouldn't be here, but having it all in one place seems useful.